src/components/Session/Chat.vue

```diff
@@ -1,79 +1,112 @@
 <template>
-  <div class="chat">
-    <div class="header">Chat</div>
-    <div class="messages-container">
-			<div class="messages">
-				<template v-for="message in messages">
-					<div class="message" v-bind:class="leftRightMessage(message)">
-						<div class="avatar" v-bind:style="message.avatarStyle"></div>
-						<div class="contents">
-              <div class="name">
-								{{message.name}}
-							</div>
-							{{message.contents}}
-              <div class="time">
-								{{message.time}}
-							</div>
+<div class="chat">
```

Comment:
Lot of stylistic changes make this PR very noisy. It's hard to tell what is changed and what is simply reformatted. Please put stylistic changes in their own PRs and avoid those changes in functional PRs

Reply:
I'll keep this in mind for future PRs, though the weird spacing is caused by a mix of tabs and spaces.

I'm going to convert all tabs to spaces and push, it looks more readable in the standardized version :smile:.